src/compas_fab/utilities/file_io.py

```diff
@@ -14,11 +14,22 @@
 def read_csv_to_dictionary(csvfile, delimiter=';'):
     """Reads a csv file and returns a dictionary with the respective keys
     specified in the first row of the csv file.
+
+    Parameters
+    ----------
+    csvfile : str
+        The path to csv file.
+    delimiter : str, optional
+        The character used to separate the values. Default `;`
```

Comment:
In rst, the monospaced font is double backticks instead of single ones (as in markdown):

```suggestion
        The character used to separate the values. Default ``;``
```